integration_test/integration_test.go

```diff
@@ -0,0 +1,322 @@
+// +build integration
+
+package integration_test
+
+import (
+	"errors"
+	"math/big"
+	"os"
+	"testing"
+	"time"
+
+	bnbExe "github.com/binance-chain/bep3-deputy/executor/bnb"
+	kavaExe "github.com/binance-chain/bep3-deputy/executor/kava"
+	"github.com/binance-chain/bep3-deputy/store"
+	"github.com/binance-chain/bep3-deputy/util"
+	"github.com/binance-chain/go-sdk/common/types"
+	bnbKeys "github.com/binance-chain/go-sdk/keys"
+	ec "github.com/ethereum/go-ethereum/common"
+	sdk "github.com/kava-labs/cosmos-sdk/types"
+	"github.com/kava-labs/go-sdk/client"
```

Comment:
pls reorg the imports

Reply:
I've formatted them with `goimports -w -local github.com/binance-chain/bep3 .`